tensorflow_addons/layers/tests/normalizations_test.py

```diff
@@ -432,14 +539,36 @@ def test_serialization(dtype):
 
 @pytest.mark.usefixtures("maybe_run_functions_eagerly")
 @pytest.mark.parametrize("dtype", [np.float16, np.float32, np.float64])
-def test_eps_gards(dtype):
+def test_eps_grads(dtype):
     set_random_seed()
-    random_inputs = np.random.rand(10, 32, 32, 3).astype(np.float32)
-    random_labels = np.random.randint(2, size=(10,)).astype(np.float32)
-    input_layer = tf.keras.layers.Input(shape=(32, 32, 3))
+
+    shape = np.random.choice(range(1, 30), np.random.randint(2, 7), replace=True)
```

Comment:
All tests will run on the same input shape and tensor. This is because `set_random_seeds()` is called at the start of every test function. 

Can you instead test for tensors ranging from 2-D to 5-D, and use randomly pick the `axis`, `channel_idx`?